src/raftstore/store/peer.rs

```diff
@@ -1037,12 +1042,16 @@ impl Peer {
                 // have no effect.
                 self.proposals.clear();
             }
+            let t = Instant::now();
             for entry in committed_entries.iter().rev() {
                 // raft meta is very small, can be ignored.
                 self.raft_log_size_hint += entry.get_data().len() as u64;
                 if lease_to_be_updated {
                     let propose_time = self.find_propose_time(entry.get_index(), entry.get_term());
                     if let Some(propose_time) = propose_time {
+                        raft_metrics
+                            .commit_log
+                            .observe(duration_to_sec(t.elapsed()));
```

Comment:
em, not t - propose_time?

Reply:
oops, mistakes when rewriting it

Reply:
oh, the result is right in your test 😭 ?

Reply:
yes, I make the rewrite to use local metrics after testing